docs/UserGuide.adoc

```diff
@@ -42,6 +42,24 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 == Features
 
 ====
+*Coming in v2.0*
```

Comment:
You may use UI mockups where necessary to describe features that are 'Coming in v2.0'.